tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -20,6 +18,56 @@ class TestPVCCreationDeletionScale(E2ETest):
     """
     Base class for PVC scale creation and deletion
     """
+    def code_inside_pvc_test(self, msg):
+        # Code that will be run inside the various pvc creation and deletion
+        # tests goes here
+        log.info(msg)
+
+    @pytest.fixture()
+    def pvc_scale_teardown(self, request):
```

Comment:
doc string needed

Reply:
I'll add doc strings in the next revision of this PR (hopefully later today)